lib/plugins/aws/deploy/lib/checkForChanges.js

```diff
@@ -134,10 +134,14 @@ module.exports = {
       // create hashes for all the zip files
       const zipFiles = globby.sync(['**.zip'], { cwd: serverlessDirPath, dot: true, silent: true });
       if (this.serverless.service.package.artifact) {
-        const arficatPath = path.join(serverlessDirPath, this.serverless.service.package.artifact);
-        zipFiles.push(arficatPath);
+        zipFiles.push(this.serverless.service.package.artifact);
       }
-      const zipFilePaths = zipFiles.map(zipFile => path.join(serverlessDirPath, zipFile));
+      const zipFilePaths = zipFiles.map(zipFile => {
```

Comment:
I think replacing `path.join` with `path.resolve` will solve it more reliably (if `zipFile` is absolute, it won't be prefixed with `serverlessDirPath`

Reply:
Updated in https://github.com/serverless/serverless/pull/7743/commits/4b94734c4e9b1535b50c70cda5a636800e2d3e79